process_rsmas.py

```diff
@@ -12,69 +12,102 @@
 import os
 import sys
 import time
+import messageRsmas
+from _process_utilities  import get_work_directory, get_project_name, send_logger
 import _processSteps as prs
```

Comment:
Can we change the abbreviation from `prs` to `psteps` or `process_steps`? Over-abbreviating is difficult to read and this variable isn't used enough to over-abbreviate